nix/pkgs/mysql80/8.0.x.nix

```diff
@@ -0,0 +1,77 @@
+{ lib, stdenv, fetchurl, bison, cmake, pkgconfig
+, boost, icu, libedit, libevent, lz4, ncurses, openssl, protobuf, re2, readline
+, zlib, zstd, numactl, perl, cctools, CoreServices, developer_cmds, libtirpc, rpcsvc-proto
```

Comment:
When I tried this locally, it failed with:

```
totten@zax:~/bknix$ nix-shell -A edge
error: anonymous function at /home/totten/bknix/nix/pkgs/mysql80/8.0.x.nix:1:1 called without required argument 'rpcsvc-proto', at /home/totten/bknix/nix/pkgs/mysql80/default.nix:9:8
(use '--show-trace' to show detailed location information)
```

I don't think that `rpcsvc-proto` is available in 20.09:

```
totten@zax:~/src/nixpkgs$ git checkout 20.09
totten@zax:~/src/nixpkgs$ grep -r rpcsvc-proto |wc
      0       0       0

totten@zax:~/src/nixpkgs$ git checkout cd63096d6d887d689543a0b97743d28995bc9bc3 
totten@zax:~/src/nixpkgs$ grep -r rpcsvc-proto |wc
      0       0       0

totten@zax:~/src/nixpkgs$ git checkout master
totten@zax:~/src/nixpkgs$ grp -r rpcsvc-proto |wc
     34     212    3205
```

Reply:
should be sorted now @totten not sure where that came from